content/docs/visualization/promdash.md

```diff
@@ -129,6 +129,19 @@ interpolated version of the given format string. To reference specific label
 values in the format string, use double curly braces: `{{label-name}}`. For
 example: `{{host}} - cluster {{cluster}}`.
 
+### Legend format filters
```

Comment:
These filters work also outside of legend format strings (for example, in expressions). So I think it would make more sense to put this into the "Using template variables" section?